### DISCUSSION TITLE
How to plot the Munsell Renotation Data?

### DISCUSSION THREAD

**Discussion by tospe** [colour-science/colour#950] [Q&A]
Hi there, 

I am the munsell notation. I am trying to explore a way of plotting colors within a munsell color wheel. 

I found this issue #88 which is still open, any ideas about what should be these plotting capabilities?

Maybe i can try to help/implement!

Thanks

**Comment by thomasmansencal** [comment]
Hi @tospe,

I think that back then I was mostly looking to plot it in such a way: 
![image](https://user-images.githubusercontent.com/99779/156901609-782992bd-b92b-4975-b291-1c296e305478.png)

Cheers,

Thomas
